### DISCUSSION TITLE
Visual Studio for Mac retired

### DISCUSSION THREAD

**Discussion by JRosanowski** [dotnet/maui#17117] [General]
https://devblogs.microsoft.com/visualstudio/visual-studio-for-mac-retirement-announcement/

> While not officially supported, we’ve also enabled rudimentary support for .NET 8 in Visual Studio for Mac for building and debugging applications. 

VS Code is taking over but it's only in Preview.

> The recently announced [C# Dev Kit](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csdevkit), [.NET MAUI](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.dotnet-maui), and [Unity](https://marketplace.visualstudio.com/items?itemName=visualstudiotoolsforunity.vstuc) Extensions for VS Code are available in preview and are intended to augment VS Code’s capabilities for .NET and C# developers. These extensions operate natively across all supported platforms, including macOS, and the experience using these will continue to be improved as they move from preview to GA and beyond.

Does anyone in the Maui team know what this all means for the .NET Maui 8 release in November? Will the current .NET 8 support in Mac VS be good enough? Will the VS Code Preview be good enough? When will VS Code be GA?

**Comment by mphill** [comment]
MAUI is even deader now.

**Comment by giuseppenovielli** [comment]
Why not use .NET MAUI to make Visual Studio 2022 for Windows to Cross-Platform solution with all features ? 😅
So we developers can write code also on tablet, ipad!

**Reply by ChaimCL** [reply]
The MSVC toolchain is hardly cross-platform, just like the Apple toolchain. Honestly VS4Mac is just called VS in name, the inner essence is still Mono Develope.

**Reply by giuseppenovielli** [reply]
Ok but for Mac, Windows and i hope Linux someday, work just now....

**Comment by JohnHDev** [comment]
This will make no difference to .NET Maui as many/most professional devs use alternatives like Rider, or VS4Windows (my preferred). I only ever switched to VS4Mac if I wanted to build Mac Catalyst to try something and hated every second of it.
VS4Mac is an appalling piece of software, badly written, and then badly rewritten with zero quality control with a manager that was not interested in fixing basic functionality (test runner anyone?).
I am not sorry to see it go and I hope that team's management comes nowhere near the .NET MAUI project. Bring the devs over, .NET MAUI needs all the help it can get. But .NET MAUI doesn't need more managers.

My 2pence.
